go.mod

```diff
@@ -12,141 +12,79 @@ require (
 	github.com/mdomke/git-semver v1.0.0
 	github.com/onsi/ginkgo/v2 v2.11.0
 	github.com/onsi/gomega v1.27.8
-	github.com/prometheus/client_golang v1.14.0
+	github.com/prometheus/client_golang v1.15.1
 	github.com/smartystreets/goconvey v1.6.4
-	github.com/stretchr/testify v1.8.0
+	github.com/stretchr/testify v1.8.1
 	go.uber.org/ratelimit v0.2.0
 	google.golang.org/grpc v1.49.0
-	k8s.io/api v0.26.4
-	k8s.io/apiextensions-apiserver v0.0.0
-	k8s.io/apimachinery v0.26.4
-	k8s.io/client-go v0.26.4
-	k8s.io/klog/v2 v2.80.1
+	k8s.io/api v0.27.2
+	k8s.io/apiextensions-apiserver v0.27.2
+	k8s.io/apimachinery v0.27.2
+	k8s.io/client-go v0.27.2
+	k8s.io/klog/v2 v2.90.1
```

Comment:
let's NOT jump to kube 1.27 just yet unless there is a clear and strong reason to, please

Reply:
probably done by mistake when I start using the controller-runtime. will fix

Reply:
The K8s update done automatically when I'm calling `go mod`  I need to figure out why

Reply:
thinking about it could be no biggie if we keep the explicit `replace` pinning deps to 1.26.7

Reply:
yes scratch it that's what we're doing

Reply:
good, false alarm! Sorry for the noise